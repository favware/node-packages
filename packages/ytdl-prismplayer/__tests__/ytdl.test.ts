import play, { play as NamedPlay } from '../src';

describe('ytdl-prismplayer', () => {
  test('should match snapshot', () => {
    expect(play).toBeDefined();
    expect(NamedPlay).toBeDefined();

    expect(play).toMatchSnapshot();
    expect(NamedPlay).toMatchSnapshot();

    expect(play).toBeInstanceOf(Function);
    expect(NamedPlay).toBeInstanceOf(Function);
  });
});
