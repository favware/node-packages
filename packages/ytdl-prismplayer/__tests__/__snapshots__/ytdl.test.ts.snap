// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ytdl-prismplayer should match snapshot 1`] = `[Function]`;

exports[`ytdl-prismplayer should match snapshot 2`] = `[Function]`;
